index.d.ts

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import * as PIXI from 'pixi.js';
+
+declare module 'react-pixi-fiber' {
+
+  /**
+   * The set of object keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
+
+  /**
+   * An object with keys in T not in U.
+   *
+   * Attribution: https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766:
+   */
+  export type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
+
+  /** The shape of an object that has an optional `children` property of any type. */
+  interface ObjectWithChildren { children?: any; }
+
+  /** The shape of `T` without it's `children` property. */
+  export type Childrenless<T extends ObjectWithChildren> = Omit<T, 'children'>;
```

Comment:
I'd rename to `Childless`.

Reply:
Ironically I had it named like that but decided it was a bad name since I'm omitting `children` not `child`, but I don't prefer it.  I will change this.